README.md

```diff
@@ -1 +1,111 @@
-# stocks-metrics-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Stock Metrics APP
+
+>This is a mobile web application that collects information from an API to provide comprehensive stocks data.
+
+![Image](./images/cover.png)
```

Comment:
- [ ] This link is broken that's why the image is not rendering. Please have a look at it.